### ISSUE TITLE
Dominating blow tooltip issue persists in latest version

### ISSUE THREAD

**Issue by fr3runn3r** [Openarl/PathOfBuilding#300]
Not sure why this is occurring, but got the initial stack trace this time. Seems to be an issue with dominating blow tooltip text, but not 100% sure. Issue only happens when changing the enlighten in the skill setup.

![image](https://cloud.githubusercontent.com/assets/26168388/25072191/69333134-22c0-11e7-98af-5dd4e7471acf.png)
[Build file.zip](https://github.com/Openarl/PathOfBuilding/files/924363/Build.file.zip)

After messing with it I saved again and was planning to try on another build and this happened:
![image](https://cloud.githubusercontent.com/assets/26168388/25072220/0959d1a4-22c1-11e7-8e51-fab619e9f014.png)

**Comment by fr3runn3r** [comment]
Trying to add dominating blow and blasphemy together on this build triggers it, it would seem.

**Comment by Openarl** [comment]
Aaah, now we're getting somewhere. I've identified the exact chain of events that's leading to this error, and it's a quite an eye opener.  

Apparently the program considers Blasphemy to be a valid support for Dominating Blow; and according to the support compatibility data for those skills it aught to be, but it obviously shouldn't be (and it isn't in-game). I'm not entirely sure what's going on here, but in the meantime I've added a check to prevent this from occurring at all. It'll be in the next update, as usual.

Thanks for your help in tracking this down; I wouldn't have stood a chance without the initial trace.
